docs/references/config/config.toml.md

```diff
@@ -0,0 +1,2055 @@
+```toml
+priv_validator_laddr = ""
+```
+
+| Value type          | string                                                     |
+|:--------------------|:-----------------------------------------------------------|
+| **Possible values** | TCP Stream socket (e.g. `"tcp://127.0.0.1:26665"`)         |
+|                     | Unix domain socket (e.g. `"unix:///var/run/privval.sock"`) |
+
+When consensus signing is outsourced from CometBFT (typically to a Hardware Security Module, like a
+[YubiHSM](https://www.yubico.com/product/yubihsm-2) device), this address is opened by CometBFT for incoming connections
+from the signing service.
+
+Make sure the port is available on the host machine and firewalls allow the signing service to connect to it.
+
+More information on a supported signing service can be found in the [TMKMS](https://github.com/iqlusioninc/tmkms)
+documentation.
+
+### node_key_file
+Path to the JSON file containing the private key to use for node authentication in the p2p protocol (more details [here](./node_key.json.md)).
+```toml
+node_key_file = "config/node_key.json"
+```
+
+| Value type          | string                                          |
+|:--------------------|:------------------------------------------------|
+| **Possible values** | relative directory path, appended to `$CMTHOME` |
+|                     | absolute directory path                         |
+
+The default relative path translates to `$CMTHOME/config/node_key.json`. In case `$CMTHOME` is unset, it defaults to
+`$HOME/.cometbft/config/node_key.json`.
+
+### abci
+The mechanism used to connect to the ABCI application.
+````toml
+abci = "socket"
+````
+
+| Value type          | string     |
+|:--------------------|:-----------|
+| **Possible values** | `"socket"` |
+|                     | `"grpc"`   |
+
+This mechanism is used when connecting to the ABCI application over the [proxy_app](#proxy_app) socket.
+
+### filter_peers
+When connecting to a new peer, filter the connection through an ABCI query to decide, if the connection should be kept.
+```toml
+filter_peers = false
+```
+
+| Value type          | boolean |
+|:--------------------|:--------|
+| **Possible values** | `false` |
+|                     | `true`  |
+
+When this setting is `true`, the ABCI application has to implement a query that returns `true` or `false` depending on
+if the ABCI application wants to keep the connection or drop it.
+
+To filter the peers by remote address, the query `/p2p/filter/addr/(peer address)` has to return `true` or `false`.
+Remote addresses have to be fully qualified, for example: `tcp://1.2.3.4:26656`.
+
+To filter the peers by [node ID](node_key.json.md), the query `/p2p/filter/id/(peer ID)` has to return `true` or `false`
+.
+
+## RPC Server
+These configuration options change the behaviour of the built-in RPC server.
+
+### rpc.laddr
+TCP or UNIX socket address for the RPC server to listen on.
+```toml
+laddr = "tcp://127.0.0.1:26657"
```

Comment:
Does it really default to `localhost` (rather than INADDR_ANY) ?

Reply:
Looks like it's the case... now that I think of it, it makes sense to indicate the user it's not safe to expose the RPC server to the public internet.

Maybe a sentence with this info would be helpful here. Up to you.

Reply:
Resolving as this info exists elsewhere in the doc